demo/horace_demo_script.m

```diff
@@ -122,45 +124,71 @@
 % function of (vector) Q and energy)
 w_sqw=sqw_eval(cc2a,@demo_FM_spinwaves,[250 0 2.4 10 5]);
 plot(w_sqw)
+keep_figure;
 %Looks vaguely like the data.
 
 %Do a fit. We'll fit parameters 1, 3 and 5 in our model, but leave 2 and 4
 %fixed (2nd vector is list of free parameters). We will
 %also have a backgound function (specified separately). The (optional) list
 %argument gives a verbose output during the fitting process
-[wfit,fitdata]=fit_sqw(cc2a,@demo_FM_spinwaves,[250 0 2.4 10 5],[1 0 1 0 1],...
-    @constant_background,[0.05],[1],'list',2,'fit',[0.001 30 0.001]);
-
+J = 250;     % Exchange interaction in meV
+D = 0;      % Single-ion anisotropy in meV
+gam = 0.5;   % Intrinsic linewidth in meV (inversely proportional to excitation lifetime)
+temp = 10;  % Sample measurement temperature in Kelvin
+amp = 5;  % Magnitude of the intensity of the excitation (arbitrary units)
+
+%[wfit,fitdata]=fit_sqw(cc2a,@demo_FM_spinwaves,[250 0 2.4 10 5],[1 0 1 0 1],...
+%    @constant_background,[0.05],[1],'list',2,'fit',[0.001 30 0.001]);
+kk = multifit_sqw (cc2a);
```

Comment:
It would be better to remove the commented out fit_sqw line so it is clear that the new parameters J...amp go with the multifit ops below along with the Do a fit commnet